GameData/RealismOverhaul/Engine_Configs/Astris_Config.cfg

```diff
@@ -51,7 +51,7 @@
 		}
 		CONFIG
 		{
-			// source: http://www.b14643.de/Spacerockets/Diverse/European_Rocket_engines/engines.htm
```

Comment:
Naw, if it doesn't work, removing it makes sense. I tried to search for a page including info on the engine on b14643, no Astris 1 info, just for 2 so far:
http://www.b14643.de/Spacerockets_1/West_Europe/Europa/Description/Frame.htm